.env.example

```diff
@@ -1,3 +1,4 @@
 # Run `docker-compose run web config generate-secret-key`
 # to get the SENTRY_SECRET_KEY value.
 SENTRY_SECRET_KEY=
+SENTRY_EMAIL_LIST_NAMESPACE=localhost
```

Comment:
I don’t think I’d add it in here since it’s such a rare setting.

Reply:
Where should we put the defaults tho as the conf file doesn't really allow that for ENV mappings?

Reply:
I’m not sure. But it needs to fallback to the same default option value. I forget all the moving pieces in play here honestly. But we can’t require every value to be explicitly set since it duplicates a lot of efforts.